CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+## Released
```

Comment:
Is this part of “Keep a Changelog”?

Reply:
Is `## Released` a part of “Keep a Changelog”?

Reply:
Fixed.